<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Soundboard</class>
 <widget class="QWidget" name="Soundboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>317</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Soundboard</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="SceneTree" name="list">
        <property name="contextMenuPolicy">
         <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolBar" name="toolbar">
        <property name="styleSheet">
         <string notr="true">QToolBar { background: transparent; }</string>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="floatable">
         <bool>false</bool>
        </property>
        <addaction name="actionAdd"/>
        <addaction name="actionRemove"/>
        <addaction name="separator"/>
        <addaction name="actionEdit"/>
       </widget>
      </item>
      <item>
       <widget class="MediaControls" name="mediaControls" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionAdd">
   <property name="icon">
    <iconset>
     <normaloff>:/res/images/plus.svg</normaloff>:/res/images/plus.svg</iconset>
   </property>
   <property name="text">
    <string>AddSound</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="class" stdset="0">
    <string>icon-plus</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/res/images/minus.svg</normaloff>:/res/images/minus.svg</iconset>
   </property>
   <property name="text">
    <string>RemoveSound</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="class" stdset="0">
    <string>icon-minus</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/settings/images/settings/general.svg</normaloff>:/settings/images/settings/general.svg</iconset>
   </property>
   <property name="text">
    <string>EditSound</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="class" stdset="0">
    <string>icon-gear</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="text">
    <string>Duplicate</string>
   </property>
  </action>
  <action name="actionFilters">
   <property name="text">
    <string>Filters</string>
   </property>
  </action>
  <action name="actionListMode">
   <property name="text">
    <string>ListMode</string>
   </property>
  </action>
  <action name="actionGridMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GridMode</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SceneTree</class>
   <extends>QListWidget</extends>
   <header>scene-tree.hpp</header>
  </customwidget>
  <customwidget>
   <class>MediaControls</class>
   <extends>QWidget</extends>
   <header>media-controls.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="obs.qrc"/>
 </resources>
 <connections/>
</ui>
